### ISSUE TITLE
Tests failing for certain specs

### ISSUE THREAD

**Issue by ananyo2012** [berkmancenter/bookanook#54]
Some tests are failing for certain specs. We need that all tests should pass.

**Comment by paulinahaduong** [comment]
@shubhpatel108 could you choose a priority level for this? Thanks! (non-urgent)

**Comment by shubhpatel108** [comment]
@ananyo2012 Can you specify which tests are failing? I'll cross check with tests failing in my local machine.

**Comment by ananyo2012** [comment]
@shubhpatel108 I see these 7 tests failing

```
Location#amenities can be an array
Location#amenities saves and refreshes as an array
NookSearch#results nooks exist filters by reservable time
NookSearch#results nooks exist filters by reservable time available for any day
Nook#set_defaults sets ExtensibleAttributes defaults
Nook#location has a location
OpenSchedule#add_9_to_5 marks all weekday blocks between 9am and 5pm local time as open
```

**Comment by shubhpatel108** [comment]
@ananyo2012 Thanks! :)

**Comment by shubhpatel108** [comment]
Solved in #97.
